### ISSUE TITLE
Add an improved Snap to Grid option

### ISSUE THREAD

**Issue by Twilight-8385** [klembot/twinejs#469]
I always use the Snap to Grid function when editing stories, but I hate having to click it everytime I open a story. Could you add an option to the main screen (the one where you see all your stories) where you can toggle it across all stories? If you're unsure of what I mean, currently it is disabled by default, the option would allow the user to toggle it to be enabled by default, so that Snap to Grid is always on unless you manually turn it off. I'm sure I'm not the only person who'd like this feature.

**Comment by janosbiroleite** [comment]
I would like that too

**Comment by jamjarae** [comment]
Another option could be to have the Snap to Grid state saved with Twine, if you turn it on, it stays on no matter if you change stories or close Twine, until you turn it off.

**Comment by Philip-Sutton** [comment]
I agree with the request.  At the least it would be good for the choice of snap to grid to be remembered persistently by stories.

**Comment by hituro** [comment]
I've always assumed that the failure of the snap to grid option to be remembered for a story was a bug. Pretty much the first thing I do every time I open a story is click a passage, have it jump out of place (because snap is now off), turn snap on, then move it back. I have to do that a lot, and it would be very nice not to have to.

**Comment by HiEv** [comment]
FYI - In v2.4.0 not only is "Snap to Grid" the default, but there doesn't seem to be any way to turn it off.  You can try the beta of it, though it still needs some work, IMHO.

**Comment by Philip-Sutton** [comment]
I just updated to Twine 2.3.16.   I don't know whether it's actually a change in Twine function but projects, set to 'snap to grid are now holding that setting within a Twine app session, as one moves between project files.

 But the memory of the 'snap to grid' settings (for projects) is not retained across twine app sessions.
